src/libs/mynewt-nimble/porting/npl/freertos/src/nimble_port_freertos.c

```diff
@@ -47,5 +47,5 @@ nimble_port_freertos_init(TaskFunction_t host_task_fn)
      * default queue it is just easier to make separate task which does this.
      */
     xTaskCreate(host_task_fn, "ble", configMINIMAL_STACK_SIZE + 600,
-                NULL, tskIDLE_PRIORITY + 1, &host_task_h);
+                NULL, 2, &host_task_h);
```

Comment:
I'm not sure it's a good idea to hard code or change these nimble tasks. Is it necessary?

Reply:
Before the change, those tasks had priorities 1 and 2. Now, they are 2 and 3.
I hard-coded the constant so it's easier to figure out the actual value of the priority. I can change them to const variables or to values relative to tskIDLE_PRIORITY as it was done before, if it improve the readability?

Reply:
I consider the other task to be "max priority" and not any number like 2 or 3, so keeping it `configMAX_PRIORITY - 1` seems more appropriate to me. The host task priority was picked by people working on nimble, and since they don't consider this host task high priority, why change it?

Reply:
Fair enough. I set the priority of the Host task higher than the other ones out of habits, but that's probably not mandatory! So I could use only 3 priorities : 0 for display, 1 for system, timer and ble host and 2 for ble LL, then!